custodial_copy_notifications.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  custodial_copy_topic_name  = "${local.environment}-dr2-cc-notifications"
+  custodial_copy_topic_arn   = "arn:aws:sns:eu-west-2:${data.aws_caller_identity.current.account_id}:${local.custodial_copy_topic_name}"
+  custodial_copy_queue_name  = "${local.environment}-dr2-cc-notifications"
+  custodial_copy_lambda_name = "${local.environment}-dr2-cc-notifications"
```

Comment:
These two should be `${local.environment}-dr2-ingest-cc-notifications`, and there variable names as well. The SNS topic is part of our CC worker and we're using it to fan-out the events to other parts of our application. The SQS queue and Lambda we're implementing at the moment are part of our ingest pipeline. We may want to implement more subscribers in future.

```mermaid
flowchart LR;
    subgraph ingest
        sqs[dr2-ingest-cc-notifications Queue] --> l[dr2-ingest-cc-notifications Lambda]--> ddb[dr2-ingest-files]
    end
    subgraph custodial-copy
        sns[dr2-cc-notifications Topic]
    end
    sns --> sqs
```